spec/ffmpeg/transcoder_spec.rb

```diff
@@ -427,5 +428,36 @@ module FFMPEG
         expect(transcoder.requires_pre_encode).to be false
       end
     end
+
+    describe "#handle_temp_files" do
+      let(:output_path) { "#{tmp_path}/output.mp4" }
+      let(:temp_output_path) { %r{#{TEMP_DIR}/output/output_.*\.mp4} }
+
+      before do
+        allow(File).to receive(:exist?).and_call_original
+        allow(FileUtils).to receive(:cp)
+        allow(FileUtils).to receive(:rm_rf)
```

Comment:
Did you need to mock these? The transcoder flow will create these files iirc so we do want them to clean up 😛